src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(({ sex, died }) => {
+    return century
+      ? Math.ceil(died / 100) === century && sex === 'm'
+      : sex === 'm';
+  });
+
+  return men.reduce((sum, { born, died }) => sum + died - born, 0)
+    / men.length;
```

Comment:
You could create the `average` function which will sum up the numbers. But it's not required